src/component/menu/Header/DrawerMenu/DrawerMenu.tsx

```diff
@@ -1,23 +1,43 @@
-import React from 'react';
+import React, { ReactNode, VFC } from 'react';
+import { Link } from 'react-router-dom';
 import { Divider, Drawer, List } from '@material-ui/core';
-import PropTypes from 'prop-types';
 import GitHubIcon from '@material-ui/icons/GitHub';
 import LibraryBooksIcon from '@material-ui/icons/LibraryBooks';
 import ExitToApp from '@material-ui/icons/ExitToApp';
-import { Link } from 'react-router-dom';
-import styles from './drawer.module.scss';
 import { ReactComponent as LogoIcon } from 'assets/icons/logoBg.svg';
-import NavigationLink from './Header/NavigationLink/NavigationLink';
-import ConditionallyRender from 'component/common/ConditionallyRender';
+import NavigationLink from '../NavigationLink/NavigationLink';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
 import { getBasePath } from 'utils/formatPath';
+import { IFlags } from 'interfaces/uiConfig';
+import { IRoute } from 'interfaces/route';
+import styles from './DrawerMenu.module.scss'; // FIXME: useStyle - theme
```

Comment:
Maybe we should make a Trello card for all the `scss` instead of the `FIXME`.

Reply:
https://trello.com/c/oiFbFdWS/591-migrate-scss-styles-to-mui